tools/bug-report/pkg/bugreport/resources.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bugreport
+
+const (
```

Comment:
We probably need a README for this command as supplement to the help message and also add some examples about expected input, output, etc

Reply:
yes, i was planning to add docs in a followup pr. probably external docs rather than too much help text here since it's for end users.